<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test10_2AClass</class>
 <widget class="QMainWindow" name="Test10_2AClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test10_2A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true">
font: 14pt &quot;宋体&quot;;</string>
      </property>
      <property name="plainText">
       <string>Using QPlainTextEdit as a Display Widget

The text is set or replaced using setPlainText() which deletes the existing text and replaces it with the text passed to setPlainText().
Text can be inserted using the QTextCursor class or using the convenience functions insertPlainText(), appendPlainText() or paste().
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="test10_2a.qrc">
     <normaloff>:/Test10_2A/Resources/filenew.png</normaloff>:/Test10_2A/Resources/filenew.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="test10_2a.qrc">
     <normaloff>:/Test10_2A/Resources/fileopen.png</normaloff>:/Test10_2A/Resources/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="test10_2a.qrc">
     <normaloff>:/Test10_2A/Resources/filesave.png</normaloff>:/Test10_2A/Resources/filesave.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test10_2a.qrc"/>
 </resources>
 <connections/>
</ui>
